RELEASE_PROCESS.rst

```diff
@@ -7,9 +7,33 @@ Make the clone using:
 
 ::
 
-     git clone --branch master git@github.com:juju/juju-gui.git
-     cd juju-gui
-     git merge origin/develop
+    git clone git@github.com:juju/juju-gui.git gui-release
+    cd juju-gui
```

Comment:
You are; where you update the changelog and then bring that over to master--it's the same way we do it in other projects, so the master commit is just the version change. The directions to checkout master and merge are further down.

If we don't care about it I can change this so it's all in master.